macros/src/lib.rs

```diff
@@ -114,6 +114,12 @@ pub fn internp(args: TokenStream) -> TokenStream {
 }
 
 /* ## Logging macros */
+#[proc_macro]
+#[proc_macro_error]
+pub fn println(args: TokenStream) -> TokenStream {
+    function_like::println::expand(args)
```

Comment:
Is there a reason you couldn't reuse `log::expand` like the other logging macros? (if so, please add a comment)

Reply:
I based it on function `write::expand` which is simlar, `println::expand` does not check feature flags and sets a different interned string. But I think both versions could be made work in `log::expand`.

Reply:
Ah, that makes sense. Yeah, would be good to merge the logic together.